src/Abp.AspNetCore/AspNetCore/Configuration/IAbpAspNetCoreConfiguration.cs

```diff
@@ -4,6 +4,7 @@
 using Abp.AspNetCore.Mvc.Results.Caching;
 using Abp.Domain.Uow;
 using Abp.Web.Models;
+using Microsoft.AspNetCore.Routing;
```

Comment:
@acjh is "using Microsoft.AspNetCore.Routing;" statement necessary ?

Reply:
Nope, I forgot to remove it in [`527a16a`](https://github.com/aspnetboilerplate/aspnetboilerplate/pull/2872/commits/527a16a38c3d5ab3b897ffd10282292d6896049c#diff-17bfa98a63ad165af72a0729fbbb024a)